docs/classes.list.js

```diff
@@ -113,51 +132,43 @@ export const borderColor = [
   'border-current',
   'border-inverted',
   'border-transparent',
-  's-border-default',
+  's-border',
   's-border-hover',
   's-border-active',
-  's-border-active-hover',
   's-border-disabled',
-  's-border-primary-default',
+  's-border-selected',
+  's-border-selected-hover',
+  's-border-inverted',
+  's-border-primary',
   's-border-primary-hover',
   's-border-primary-active',
-  's-border-primary-active-hover',
-  's-border-primary-subtle-default',
+  's-border-primary-subtle',
   's-border-primary-subtle-hover',
   's-border-primary-subtle-active',
-  's-border-primary-subtle-active-hover',
-  's-border-positive-default',
+  's-border-positive',
   's-border-positive-hover',
   's-border-positive-active',
-  's-border-positive-active-hover',
-  's-border-positive-subtle-default',
+  's-border-positive-subtle',
   's-border-positive-subtle-hover',
   's-border-positive-subtle-active',
-  's-border-positive-subtle-active-hover',
-  's-border-negative-default',
+  's-border-negative',
   's-border-negative-hover',
   's-border-negative-active',
```

Comment:
We also added `s-border-negative-selected` and `s-border-negative-selected-hover` in [@warp-ds/css@1.0.0-alpha.36](https://github.com/warp-ds/css/commit/1d3bfdb0787eedcff16782099c5657138f607826#diff-2704f7af6af729d92f7b4b60741f5e6e2c8bfbedf35b98f0aaae6aa1ceca1570)